toolkit/scripts/srpm_pack.mk

```diff
@@ -38,23 +38,25 @@ $(BUILD_SRPMS_DIR): $(STATUS_FLAGS_DIR)/build_srpms.flag
 	@echo Finished updating $@
 
 ifeq ($(DOWNLOAD_SRPMS),y)
+
+.SILENT: $(STATUS_FLAGS_DIR)/build_srpms.flag
```

Comment:
+1 for reducing the noise. But instead of discarding the output, please redirect it into a log file, so that we have something to look at when we need to debug. There's an example of how that could be done in `toolchain.mk` at the very bottom. The last `$(toolchain_rpms)` target.